src/main/java/org/prebid/server/bidder/huaweiads/HuaweiAdSlotBuilder.java

```diff
@@ -96,8 +100,10 @@ private AdSlot30 makeNativeAdSlot(Native xNative) {
         final long numImage = assets.stream().map(Asset::getImg).filter(Objects::nonNull)
                 .filter(img -> IMAGE_ASSET_TYPE_MAIN.equals(img.getType()))
                 .count();
+        final List<Format> formats = makeFormatListForNative(assets, numImage);
         return AdSlot30.builder()
                 .detailedCreativeTypeList(makeDetailedCreativeTypeList(numVideo, numImage))
+                .format(CollectionUtils.isEmpty(formats) ? null : formats)
```

Comment:
according to the PBS Go empty list should be passed to the adslot

Reply:
Ok. Got it.